.example.env

```diff
@@ -1,4 +1,4 @@
-SUPABASE_ANON_KEY=xxx
-SUPABASE_URL=https://xxx.supabase.co
+SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24iLCJleHAiOjE5ODM4MTI5OTZ9.CRXP1A7WOeoJeXxjNni43kdQwgnWNReilDMblYTn_I0
```

Comment:
Where'd this value come from? Does it work on local Supabase instances?

Reply:
Yes correct, this is the local development credentials. It's being outputted when running `supabase start` or `supabase status` (when local stack is running already)